package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "jitsi-meet-web-sdk",
+  "version": "0.0.0",
+  "description": "Web SDK for the Jitsi Meet IFrame",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
+    "lint": "eslint -c .eslintrc.js --ext .ts \"src\"",
+    "postversion": "git push && git push --tags",
```

Comment:
Please drop this, raising the version should not push automatically. We'll use PRs.